src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,19 @@
-export const CommentInfo = () => <>Put the comment here</>;
+import React from 'react';
+
+export const CommentInfo = ({ comment }) => {
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{comment.name}</strong>
+
+        {' by '}
+
+        <a className="CommentInfo__email" href={`mailto:${comment.email}`}>
+          {comment.email}
```

Comment:
Use destruction for `comment`

Reply:
isn't fixed. 
Read about object destructing [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)